CHANGELOG.md

```diff
@@ -7,13 +7,13 @@
 * Update Razor to 9.0.0-preview.24327.6 (PR: [#7273](https://github.com/dotnet/vscode-csharp/pull/7273))
   * Use a named pipe to communicate projectinfo in vscode (PR: [#10521](https://github.com/dotnet/razor/pull/10521))
   * Reduce allocations in Razor's DirectiveVisitor (PR: [10521](https://github.com/dotnet/razor/pull/10521))
-* Update Roslyn to 4.12.0-1.24359.8 (PR: [#7273](https://github.com/dotnet/vscode-csharp/pull/7273))
+* Update Roslyn to 4.12.0-1.24359.11 (PR: [#7326](https://github.com/dotnet/vscode-csharp/pull/7326))
+  * Fix issue causing error toasts to display on diff window views or new C# documents (PR: [#74300](https://github.com/dotnet/roslyn/pull/74300))
+  * Fix issue where loaded projects would be missing references (PR: [#74189](https://github.com/dotnet/roslyn/pull/74189))
   * Fix UseNullPropagationCodeFixProvider for parenthesized property access (PR: [#74316](https://github.com/dotnet/roslyn/pull/74316))
-  * Sync solution contents consistently(PR: [#72860](https://github.com/dotnet/roslyn/pull/72860))
```

Comment:
❓ how do I know which PRs to include and which not to? Is it just manually looking at the PR?

Reply:
yeah essentially :(.  Probably need to figure out a better automated sln like maybe tags on PRs that affect VSCode